tests/client/test_client_task_meilisearch.py

```diff
@@ -91,3 +92,30 @@ def test_get_task_inexistent(client):
     """Tests getting a task that does not exists."""
     with pytest.raises(Exception):
         client.get_task("abc")
+
+
+def test_cancel_tasks(client):
+    """Tests cancel a task with uid 1."""
+    task = client.cancel_tasks({"uids": ["1", "2"]})
+    tasks = client.get_tasks({"types": "taskCancelation"})
+
+    assert isinstance(task, TaskInfo)
+    assert task.task_uid is not None
+    assert task.index_uid is None
+    assert task.status in {"enqueued", "processing", "succeeded"}
```

Comment:
This is only relevant if you wait_for_task and the check if the status is `succeeded`. Then we ensure that you made a correct request to Meilisearch.

Reply:
It is not possible that a return of task is already `succeeded`?

Reply:
Yes, but by doing wait_for_task we are sure it is succeeded! And not something else